assets/javascripts/discourse/routes/subscribe-index.js

```diff
@@ -5,4 +5,23 @@ export default Route.extend({
   model() {
     return Product.findAll();
   },
+
+  afterModel(products) {
+    if (products.get("length") === 1) {
```

Comment:
I think all those `get()`s can be replaced with native getters? (e.g. `products.length`)

Reply:
Sure, that works.  Are you recommending this in this specific case where the native getters work just fine or do we have some special handling where it works for more complex Ember objects?